content/docs/command-reference/repro.md

```diff
@@ -10,12 +10,23 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-c <path>] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
+                 [--no-commit] [--downstream] [--pull] [--glob]
                  [targets [targets ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets        Stage or path to dvc.yaml or .dvc file to reproduce.
```

Comment:
This does not look good. Can we move this content to a separate section, for example?
> #### The `target` option

But, I'd like to keep information about the ":" addressing though.

Reply:
Yes! This is good, @skshetry - my 2cs - it deserves a section similar to the `### Parallel execution` below (and it should be before) ... and there should be a clear link to it in the first paragraph of the description ... or even in the synopsis?

Reply:
I think it would be better to shift **Parallel execution** to the **Examples** section or further below and add a one-liner linking to it in the **Description**. 

For now, we could just add it to the Synopsis, but in the long run, we could move this `targets` section in the `Options` itself as done on [`git add`](https://git-scm.com/docs/git-add#Documentation/git-add.txt-ltpathspecgt82308203).

Reply:
> I think it would be better to shift Parallel execution to the Examples

it might be, but let's not touch it in this PR - it's not related at the moment to my mind.

> For now, we could just add it to the Synopsis

you mean link or the whole text?

Reply:
> you mean link or the whole text?

I meant all of the text in Synposis, but I could also just add a link on the synopsis (it's not possible to link in the `usage`, is it?), and move the text to the **Options**. What do you think?

Reply:
> it's not possible to link in the usage, is it

probably not at the moment :(

I would for now create a section in the description like you suggested. Options works fine as well. And let's create a ticket to figure out a better way to link things from the synopsis to their actual location.